packages/network-enablement-controller/tsconfig.build.json

```
@@ -0,0 +1,20 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist",
+    "rootDir": "./src"
+  },
+  "references": [
+    {
+      "path": "../../packages/base-controller/tsconfig.build.json"
+    },
+    {
+      "path": "../../packages/network-controller/tsconfig.build.json"
+    },
+    {
+      "path": "../../packages/multichain-network-controller/tsconfig.build.json"
+    }
+  ],
```

Nit: What do you think about ? also is there a reason why controller-utils was not included ?
```suggestion
  "references": [
    { "path": "../base-controller/tsconfig.build.json" },
    { "path": "../network-controller/tsconfig.build.json" },
    { "path": "../multichain-network-controller/tsconfig.build.json" },
    { "path": "../controller-utils/tsconfig.build.json" }
  ],